vaadin-login-flow-parent/vaadin-login-flow-integration-tests/src/test/java/com/vaadin/flow/component/login/tests/AbstractLoginIT.java

```diff
@@ -16,42 +16,24 @@
 package com.vaadin.flow.component.login.tests;
 
 import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.Ignore;
 
 import com.vaadin.flow.component.button.testbench.ButtonElement;
 import com.vaadin.flow.component.login.testbench.LoginFormElement;
 import com.vaadin.flow.component.textfield.testbench.PasswordFieldElement;
 import com.vaadin.flow.component.textfield.testbench.TextFieldElement;
-import com.vaadin.testbench.parallel.BrowserUtil;
-import com.vaadin.tests.AbstractParallelTest;
+import com.vaadin.tests.AbstractComponentIT;
 
-public abstract class BasicIT extends AbstractParallelTest {
-
-    @Before
-    public void init() {
-        String url = getBaseURL().replace(super.getBaseURL(),
-                super.getBaseURL() + "/vaadin-login");
-        getDriver().get(url);
-    }
-
-    public abstract LoginFormElement getLoginForm();
-
-    @Test
-    public void testDefaults() {
-        LoginFormElement login = getLoginForm();
+@Ignore
+public abstract class AbstractLoginIT extends AbstractComponentIT {
```

Comment:
Renamed this file from `BasicIT` to `AbstractLoginIT` to align with other components e.g. `ComboBox`.